testsuite/framework/src/main/java/org/keycloak/client/testsuite/framework/KeycloakVersion.java

```diff
@@ -0,0 +1,16 @@
+package org.keycloak.client.testsuite.framework;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * @author <a href="mailto:mposolda@redhat.com">Marek Posolda</a>
```

Comment:
Minor: Isn't it rather you, who is author of this class? :-) As it seems this is completely new class, which is not copyied from Keycloak server codebase?

Reply:
Also copy paste typo